Ag s3 archive

agoude opened the pull request:
- New guide(s)
- Update of existing guide(s)

## Description

We will deploy Cold Archive v2 (object-level granularity) by Nov 19th. To do so, we are creating one dedicated guide and up
## Mandatory information

The translations in this Pull Request have been done using:
- Not done yet for FR.

<!-- Delete the line which doesn't apply: -->
- This Pull Request shouldn't be merged before: 2025-11-19

<!-- If you know about the following, please mention it. If you don't know, delete the line.-->
- This Pull Request content should be replicated for the US OVHcloud documentation : YES if applicable

agoude committed:
Create new repo Restoring guide.en-gb.md

New folder for a "Restoring an Object Storage Object from Cold Archive storage class" guide

agoude committed:
Update EN restoring objects guide.en-gb.md

v0.1 Restoring Objects guide

agoude committed:
Create meta.yaml

meta.yaml file creation

agoude committed:
Update index.md

Index update for public-cloud-storage-object-storage-restoring-objects >> Object Storage - Restoring an archived object from Cold Archive storage class

agoude committed:
Update Object Storage classes guide.en-gb.md

- new Cold Archive tier details
- html table update instead of markdown
- new additional details for Cold Archive billing example

agoude committed:
Update guide.en-gb.md

- Adding warning message for new Cold Archive

agoude committed:
Update guide.en-gb.md

- Adding a warning message for Cold Archive (object-level)

agoude committed:
Update guide.en-gb.md

- Adding a warning message for Cold Archive (object-level)

lason-ovh commented on pages/storage_and_backup/object_storage/s3_choosing_the_right_storage_class_for_your_needs/guide.en-gb.md:
```diff
@@ -64,6 +119,12 @@ It refers to the ability to **automatically manage the storage lifecycle of your
 
+#### Cold Archive (Object-level management) class considerations
+
+When using the Cold Archive class in an Object Storage general-purpose bucket, **archived objects are not available in real-time**. The user is asked to first restore the object through a "restore" request and then, after some time (from minutes up to multiple hours), the object is available to download for a duration set by the user when requesting it and is billed upfront at the Standard class  ...
```
maybe add precision about restore fees composition i.e for each object restored, a one time restoration fee per GiB of data plus the storage cost at the Standard rate of the object

lason-ovh commented on pages/storage_and_backup/object_storage/s3_restoring_objects/guide.en-gb.md:
````diff
@@ -0,0 +1,69 @@
+>>
+> Via the OVHcloud API
+>> ```bash
+>> ADDING API ROUTE FOR RESTORATION ON COLD ARCHIVE
````
/cloud/project/{serviceName}/region/{regionName}/storage/{name}/object/{key}/restore

lason-ovh commented on pages/storage_and_backup/object_storage/s3_restoring_objects/guide.en-gb.md:
```diff
@@ -0,0 +1,69 @@
+> Via the OVHcloud Control Panel
+>> Click on `Object Storage`{.action} in the navigation bar on the left and then on the `My containers`{.action} tab.
+>>
+>> In the list of objects, click on the `...`{.action} button associated to the object you want to restore, and then click `Restore`{.action}.
```
add a section to monitor restoration status via the UI in the object listing: manager will add a restoration status for all objects in deep archive

lason-ovh reviewed (changes requested):
cf comments

agoude committed:
Update guide.en-gb.md

- adding details on class considerations (restoration fee composition)

agoude committed:
Update guide.en-gb.md

- Adding ovhcloud api v1 route for restore

agoude committed:
Update guide.en-gb.md

- adding sentence for monitoring from the OVHcloud Control Panel

agoude committed:
Update guide.en-gb.md

endpoints v2 part 1

agoude committed:
Update guide.en-gb.md

- update class mapping between S3 api and OVHcloud tiers
- update geo availability and endpoints

agoude commented:
@lason-ovh I added all your comments and also added modification to the s3_locations doc

lason-ovh reviewed (approved):
ok for me

agoude committed:
Update guide.en-gb.md

- correct tabs alignement

agoude committed:
Create guide.fr-fr.md

changelog:
- create FR version of the restore object guide

agoude committed:
Update guide.fr-fr.md

- Adding details about Cold Archive, new class of Object Storage

agoude committed:
Update guide.fr-fr.md

Changelog
Adding new Cold Archive details in regions and class mapping in FR

agoude committed:
Update guide.fr-fr.md

change date

agoude committed:
Update guide.fr-fr.md

Adding header message

agoude committed:
Update guide.fr-fr.md

Adding header message

agoude committed:
Update guide.fr-fr.md

Adding header message

bs-ovh committed:
EN proof

benchbzh committed:
first proofreading & date update

benchbzh committed:
first proofreading & date update

benchbzh committed:
first proofreading & date update

benchbzh committed:
minor update

benchbzh committed:
minor update

benchbzh committed:
duplication

benchbzh commented:
Thank you for your contribution @agoude!
